### PR TITLE
QA: Update dependencies

### PR DESCRIPTION
**Pull request by michalbundyra**
|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

- use spatie/yaml-front-matter instead of mnapoli/front-yaml
  (so now we can use also symfony/yaml 5.0)
- add replace laminas/laminas-zendframework-bridge as we are not using
  any not migrated components
- add platform configuration into composer.json

Replaces #45

### CODE REVIEW

File: src/Security/Console/BuildCommand.php [hunk 1]
```diff
@@ -36,13 +32,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         }
 
         $io->writeln('<info>Building advisory list</info>');
-        $env = Environment::createCommonMarkEnvironment();
-        $env->addExtension(new TableExtension());
```
Comment (weierophinney) [inline src/Security/Console/BuildCommand.php:40]: The one thing I see missing from this patch is that you haven't setup the CommonMark environment to add the table extension - which is not enabled by default, and which we will likely use. That needs to be part of the setup in the Document implementation.

Comment (michalbundyra) [reply src/Security/Console/BuildCommand.php:40]: Yeah, I was thinking about it, but I see we are not using it anywhere. And it was enabled only in security build command, not anywhere else.

I've noticed that we have also `postProcessHtml` in `AbstractCollection` which somehow process tables and I wasn't sure why it's there as the extension was not enabled. Maybe everything we should move to the document? (I mean enable the table extension and process html tables to add additional classes?)

Comment (weierophinney) [reply src/Security/Console/BuildCommand.php:40]: > we are not using it anywhere

We used it in blog posts as recently as the last on on the ZF site. I think we should have it enabled by default, so we don't have to add it later, when we inevitably want to add a table.

I think the post processing of tables was to ensure they work correctly with Bootstrap.

Comment (michalbundyra) [reply src/Security/Console/BuildCommand.php:40]: @weierophinney TableExtension and post processing added to the Document.

**Pull request merged**